api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -1057,7 +1057,6 @@
         "type": "object"
       },
       "io.k8s.api.core.v1.Container": {
-        "description": "A single application container that you want to run within a pod.",
```

Comment:
I don't underatand why this LOST a comment?

Reply:
Yes, I saw it and also surprised. I think it is because the type alias `InitContainer` doesn't have a comment. I can try to fix it.